R/scale-linetype.r

```diff
@@ -14,6 +14,15 @@
 #' base + geom_line(aes(linetype = variable))
 #'
 #' # See scale_manual for more flexibility
+#'
+#' # show common line types
+#' df_lines <- data.frame(linetype = factor(1:4, labels = c("solid", "longdash", "dashed", "dotted")))
```

Comment:
A character vector would order the types alphabetically, rather than by spatial freq/luminance. Here's an alternative:

``` r
linetypes <- c("solid", "longdash", "dashed", "dotted")
df_lines <- data.frame(value = seq_along(linetypes))
ggplot(df_lines) +
  geom_hline(aes(linetype = factor(value), yintercept = value), size = 2) +
  geom_text(aes(x = 0, y = value+.25, label = linetypes[value])) +
  scale_linetype_manual(values = linetypes, guide = 'none') +
  theme_void()
```
![image](https://cloud.githubusercontent.com/assets/2257540/18030219/05c0a94a-6c74-11e6-85b5-0e99a1ba508a.png)